examples/sample_program.py

```diff
@@ -486,10 +486,13 @@ def parse_images(content):
     return content
 
 
-
+chef = SampleChef()
 if __name__ == '__main__':
     """
     This code will run when the sushi chef is called from the command line.
     """
-    chef = SampleChef()
```

Comment:
I think it makes more sense for the `chef` instance to be created inside the `__main__` block as before, but it should work either way.